src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -73,9 +73,27 @@ class AnonymousElectorDocument(
     @field:Size(max = 80)
     var gssCode: String,
 
+    @field:NotNull
+    @field:Size(max = 7)
+    var electoralRollNumber: String,
+
+    @field:NotNull
+    @field:Size(max = 255)
+    var aedTemplateFilename: String,
+
+    @field:NotNull
+    var issueDate: LocalDate,
+
+    @field:NotNull
+    var requestDateTime: Instant,
```

Comment:
Is this the same as `dateCreated`? If so, do we need it?

Reply:
This would be set during mapping, instead of waiting for the JPA to populate it whilst persisting - this means it can be set at construction.  Also, it records when request is made, whereby the `created` is for audit trail.  It's similar to way we have both `userId` and `createdBy` properties.  Lastly, this is only moving from one table to another - so hasn't been altered by the refactor.

Reply:
Thanks for the explanation. IMHO it feels a bit unnecessary, but I'm not going to worry about it 👍